staging/src/k8s.io/noderesourcetopology-api/manifests/crd-instance-example.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: topology.node.k8s.io/v1alpha1
+kind: NodeResourceTopology
+metadata:
+  name: node1
+topologyPolicies:
+  - single-numa-node
```

Comment:
Constants should be CamelCaseWords

Reply:
The policy name here corresponds to Topology Manager policy as can be seen [here](https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/#topology-manager-policies).

Reply:
Ack, will update!